### ISSUE TITLE
elvis_style:is_otp_module only checks `behavior` attribute, not `behaviour`

### ISSUE THREAD

**Issue by NAR** [inaka/elvis#550]
In Erlang both the US and British spelling is accepted for the `-behaviour` attribute, however, the `elvis_style:is_otp_module/1` function only checks for the US spelling and doesn't detect OTP behaviours using the British spelling:

```
    IsBehaviorAttr = fun(Node) -> behavior == ktn_code:type(Node) end,
```

Maybe use something like this:
```
    IsBehaviorAttr = fun(Node) -> behavior == ktn_code:type(Node) orelse behaviour == ktn_code:type(Node) end,
```

**Comment by elbrujohalcon** [comment]
I would've expected `ktn_code` to return `behavior` in both cases, tho… 🤔 
If you're certain that it doesn't, and you feel like submitting a PR with the proposed change, that would be highly appreciated.

**Comment by NAR** [comment]
When I changed the attribute to `-behaviour`, I didn't get warning - I presume because the code doesn't detect that the module is an OTP behaviour. WIth `-behavior` I got the warning. I'll try to cook up a PR.

**Comment by NAR** [comment]
See https://github.com/inaka/elvis_core/pull/139

**Comment by paulo-ferraz-oliveira** [comment]
I think I've touched this subject somewhere, and @jfacorro got involved. Let me do a quick search...

**Comment by paulo-ferraz-oliveira** [comment]
Related: https://github.com/inaka/elvis/issues/490.

**Comment by paulo-ferraz-oliveira** [comment]
> I think I've touched this subject somewhere, and @jfacorro got involved. Let me do a quick search...

My issue was not related to this, but it also boiled down to spelling 😄: https://github.com/inaka/elvis/issues/522.

**Comment by elbrujohalcon** [comment]
Fixed with inaka/elvis_core#139
